packages/website/babel.config.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   presets: ['next/babel', '@babel/typescript'],
   plugins: [
+    'emotion',
     [
       'styled-components',
```

Comment:
Should this be removed?

Reply:
Not yet. We will still have a bunch of SC code on our site for some time - until AK updates and we update AK.

Reply:
But surely the AK components are already transpiled. Isn't this SC babel plugin only for our own stuff?

Reply:
I'll try removing it and let you know.

Reply:
I removed it and everything broke, so leaving it in for now.

Reply:
this makes me angery